Dockerfile

```diff
@@ -1,15 +1,11 @@
 FROM python:2.7.11
 
-MAINTAINER Robert Kaye <rob@metabrainz.org>
```

Comment:
Didn't actually remove it. It's just the file got moved and diffs here are displayed differently. https://github.com/metabrainz/listenbrainz-server/pull/94/files#diff-0e9722a692af2b244674b56146a1ecd3R3

I don't think it's important in our case. Everyone is working on these files anyway.